crates/arroyo-rpc/default.toml

```diff
@@ -82,3 +82,7 @@ user = "arroyo"
 password = "arroyo"
 
 [logging]
+format = "plaintext" # option: plaintext / json / logfmt 
+nonblocking = true
```

Comment:
The current default is actually "nonblocking = false" since nonblocking mode causes a large increase in memory usage

Reply:
"nonblocking = false" will make logging as sync process. if it cause large memory usage. I suggest to decrease `buffered_lines_limit` of `tracing-appender` like below code (default is `pub const DEFAULT_BUFFERED_LINES_LIMIT: usize = 128_000`)
https://github.com/tokio-rs/tracing/blob/527b4f66a604e7a6baa6aa7536428e3a303ba3c8/tracing-appender/src/non_blocking.rs#L395-L398